src/storm-pars/transformer/RobustParameterLifter.h

```diff
@@ -0,0 +1,182 @@
+#pragma once
+
+#include <memory>
+#include <set>
+#include <unordered_map>
+#include <vector>
+
+#include "storm-pars/analysis/Order.h"
```

Comment:
With some forward declarations, we can probably reduce compile times significantly here.